### PR TITLE
VMware: new module: vmware_guest_custom_attribute_defs

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/vmware/vmware_guest_custom_attribute_defs.py
test/integration/targets/vmware_guest_custom_attribute_defs/aliases
test/integration/targets/vmware_guest_custom_attribute_defs/tasks/main.yml

##### ANSIBLE VERSION

```
2.6devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
@pdellaert @tchernomax Could you please review this ?

File: lib/ansible/modules/cloud/vmware/vmware_guest_attribute_defs.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
```
Comment (pdellaert) [inline lib/ansible/modules/cloud/vmware/vmware_guest_attribute_defs.py]: Same as PR #38114, potentially rename it to vmware_guest_custom_attribute_defs, or vmware_guest_custom_attr_definitions

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_guest_attribute_defs.py]: Make sense.

**Comment by Akasurde** [comment]
@pdellaert Done.

**Review by pdellaert — APPROVED** [review]

**Pull request merged**